crates/driver/src/infra/blockchain/token.rs

```diff
@@ -134,29 +143,51 @@ impl Erc20 {
                 })
                 .collect(),
         );
+        let mut delegate_call = self
+            .ethereum
+            .contracts()
+            .settlement()
+            .simulate_delegatecall(
+                balance_helper.address(),
+                ethcontract::Bytes(balance_call.tx.data.unwrap_or_default().0),
+            )
+            .from(SIMULATION_ACCOUNT.clone());
+
         // Create the access list for the balance simulation only if they are enabled
         // system-wide.
         if !disable_access_lists {
-            let access_list_call = contracts::storage_accessible::call(
-                method.tx.to.unwrap(),
-                contracts::bytecode!(contracts::support::Balances),
-                method.tx.data.clone().unwrap(),
-            );
+            let access_list_call = CallRequest {
+                data: delegate_call.tx.data.clone(),
+                ..Default::default()
+            };
             let access_list = self
                 .ethereum
                 .create_access_list(access_list_call)
                 .await
+                .tap_err(|err| {
+                    tracing::error!(?err, "failed to create access list for balance simulation");
```

Comment:
Not being able to generate the access lists is usually not a big deal. Best to avoid using the `error` or `warn` log level for non-problematic things.
```suggestion
                    tracing::debug!(?err, "failed to create access list for balance simulation");
```

Reply:
Yeah, it is just a leftover from my local runs. Thanks.